src/commands/synthetics/batch.ts

```diff
@@ -211,7 +211,10 @@ const getResultFromBatch = (
 
   const pollResult = pollResultMap[resultInBatch.result_id]
 
-  if (hasTimedOut) {
+  if (safeDeadlineReached) {
+    pollResult.result.failure = {code: 'SAFE_DEADLINE_REACHED', message: 'Datadog did not answer on time.'}
```

Comment:
I see that this "Datadog did not answer on time" message is duplicated between two places. Is it possible to have these messages in a single place?